pom.xml

```diff
@@ -161,11 +168,21 @@
     </testResources>
     <pluginManagement>
       <plugins>
+        <plugin>
```

Comment:
I'm not sure why we need this

Reply:
You are right, we probably don't. Let me make sure

Reply:
Changed to failsafe which we actually use